scripts/apps/desks/views/desk-config-modal.html

```diff
@@ -161,15 +161,13 @@ <h3 class="modal__heading" ng-show="desk.edit._id" translate>Edit "{{desk.edit.n
                             </div>
                             <div class="sd-line-input sd-line-input--is-select">
                                 <select
+                                    ng-options="template._id as template.template_name for template in templates"
```

Comment:
You could have left `<option ng-repeat="template in templates" ...` I was only talking about `data-test-id` attribute. This is fine too though.